README.md

```diff
@@ -27,7 +27,7 @@ or
 import {subscribe} from 'core/pubsub';
 import RealTimeEvents from 'tool_realtime/events';
 
-subscribe(RealTimeEvents.EVENT, (context, component, area, itemid, payload) => {
+subscribe(RealTimeEvents.EVENT, (eventData) => {
```

Comment:
because this is simple now, in the lines below can you gives examples of each attribute, roughly something like

```
// eventData.context = 123
// eventData.component = "tool_foobar";
// eventData.area = "mychannel";
// eventData.itemid = 12345;
// eventData.payload = { foo: "bar" }
```